vuepress/docs/next/tutorials/micro-frontends/react.md

```diff
@@ -216,12 +216,12 @@ Example:
 PUBLIC_URL=http://quickstart-entando.192.168.64.34.nip.io/entando-de-app/cmsresources/my-widget
 ```
 
-3. Replace `quickstart-entando.192.168.64.34.nip.io/app-builder` with the URL for your Entando App Builder instance. → [How to find your Entando App Builder URL.](../../docs/getting-started/#deploy-entando)
+3. Replace `quickstart-entando.192.168.64.34.nip.io/entando-de-app` with the URL for your Entando application.
```

Comment:
This (and the line above it) are incorrect. We should be using a relative path there or passing a path as an argument to the MFE unless this is for local dev.

Reply:
Good call. I never liked that this broke running the app locally too. Will update to .env.production with a relative url.

Reply:
Updated